_includes/fe-plugin-intro.md

```diff
@@ -0,0 +1,17 @@
+Frontend plugins provide a way to change the behavior of Deck, Spinnaker's UI
+service. You can add configuration and validation for new stages provided by
+Orca plugins, override existing components with your own implementation, or add
+new Kind definitions for custom resources in your environment.
```

Comment:
I might say `Kubernetes Kind` for those that aren't kubernetes-centric.

Reply:
- [ ] Kind -> K8s Kind